ui/components/app/collectibles-items/collectibles-items.js

```diff
@@ -0,0 +1,225 @@
+}) {
+  const t = useI18nContext();
+  const defaultDropdownState = {};
+  const ipfsGateway = useSelector(getIpfsGateway);
+
+  Object.keys(collections).forEach((key) => {
+    defaultDropdownState[key] = true;
+  });
+  const history = useHistory();
+
+  const [dropdownState, setDropdownState] = useState(defaultDropdownState);
+  return (
+    <div className="collectibles-items">
+      <Box padding={[6, 4]} flexDirection={FLEX_DIRECTION.COLUMN}>
+        <>
+          {Object.keys(collections).map((key, index) => {
+            const {
+              collectibles,
+              collectionName,
+              collectionImage,
+            } = collections[key];
+
+            const isExpanded = dropdownState[key];
+            return (
+              <div
+                className="collectibles-items__item"
+                key={`collection-${index}`}
+                onClick={() => {
+                  setDropdownState((_dropdownState) => ({
+                    ..._dropdownState,
+                    [key]: !isExpanded,
+                  }));
+                }}
+              >
+                <Box
+                  marginBottom={2}
+                  display={DISPLAY.FLEX}
+                  alignItems={ALIGN_ITEMS.CENTER}
+                  justifyContent={JUSTIFY_CONTENT.SPACE_BETWEEN}
+                  className="collectibles-items__item__accordion-title"
+                >
+                  <Box
+                    alignItems={ALIGN_ITEMS.CENTER}
+                    className="collectibles-items__item__collection-header"
+                  >
+                    {collectionImage ? (
+                      <img
+                        src={collectionImage}
+                        className="collectibles-items__item__collection-image"
+                      />
+                    ) : (
+                      <div className="collectibles-items__item__collection-image-alt">
+                        {collectionName[0]}
+                      </div>
+                    )}
+                    <Typography
+                      color={COLORS.BLACK}
+                      variant={TYPOGRAPHY.H5}
+                      margin={[0, 0, 0, 2]}
+                    >
+                      {`${collectionName} (${collectibles.length})`}
+                    </Typography>
+                  </Box>
+                  <Box alignItems={ALIGN_ITEMS.FLEX_END}>
+                    <i
+                      className={`fa fa-chevron-${
+                        isExpanded ? 'down' : 'right'
+                      }`}
+                    />
+                  </Box>
+                </Box>
+                {isExpanded ? (
+                  <Box display={DISPLAY.FLEX} flexWrap={FLEX_WRAP.WRAP} gap={4}>
+                    {collectibles.map((collectible, i) => {
+                      const {
+                        image,
+                        address,
+                        tokenId,
+                        backgroundColor,
```

Comment:
Does `backgroundColor` work? Is it supposed to come from the opensea api? I notice they have a background color on their displays.

<img width="830" alt="Screen Shot 2021-12-11 at 8 28 43 AM" src="https://user-images.githubusercontent.com/8112138/145630808-97a44d3a-6670-466d-b838-b6fcbbdda0a2.png">


<img width="571" alt="Screen Shot 2021-12-11 at 8 28 39 AM" src="https://user-images.githubusercontent.com/8112138/145630794-0e95bee8-a751-4a61-8c06-b1011a3f3e17.png">

Reply:
hmmm this should be working. Let me take a look!

Reply:
@georgewrmarshall could you test this again?

Reply:
Looks like the backgroundColor is missing the `#` so it's not showing up. Updated to include the hex and it worked.

<img width="527" alt="Screen Shot 2021-12-14 at 10 40 41 AM" src="https://user-images.githubusercontent.com/8112138/145893259-d81aa30a-f877-4b48-99fd-233ae2bb465c.png">

<img width="712" alt="Screen Shot 2021-12-14 at 10 38 48 AM" src="https://user-images.githubusercontent.com/8112138/145893286-36f35594-be31-4197-87de-20427046e84c.png">

Might be a bit more challenging on the details view as the `Card` component wraps the `Box` component that only allows a set of background colors. The solution here is to allow the `style` prop on the `Box` component which, it should allow anyway.